flixel/input/gamepad/id/LogitechID.hx

```diff
@@ -7,6 +7,12 @@ import flixel.input.gamepad.FlxGamepad;
  */
 class LogitechID
 {
+	public static inline var SUPPORTS_MOTION = false;
```

Comment:
Not quite. Motion and Pointer are two different things, at least based on how I've got it set up currently.

Motion is pitch/roll/yaw detection -- So far in all my tests the WiiMote supports pitch/roll, and the PS4/PS3 also support pitch/roll, but only the WiiMote reports them in a way I can reliably detect without crazy custom driver mappings.

The WiiMote also supports an Infrared camera which is what it uses to drive the on-screen pointer on the Wii/WiiU. Basically the infrared camera and the sensor bar combine to give you X/Y coordinates, and then you can multiply the (0.0-1.0),(0.01-1.0) X/Y values to get a pointer value.

The PS4 also has a touchpad, as does the Vita, the 3DS, the WiiU gamepad, and the Ouya, and so the way I thought of this, was that the WiiMote IR camera pointer has more in common with these touchpads than it does with "motion" as it just spits out analog X/Y coordinates, so we should have a separate "pointer" API for all of those.

As of right now I have only been able to get the WiiMote to be able to actually use the pointer API on the PC, but on the native console side all of the above could fill out the pointer API.